lib/trino-orc/src/main/java/io/trino/orc/OrcWriter.java

```diff
@@ -467,6 +467,8 @@ public void close()
         try (Closeable ignored = orcDataSink) {
             flushStripe(CLOSED);
         }
+
+        bufferedBytes = 0;
```

Comment:
why is it not reset by `flushStripe`?

Reply:
because OrcWriter accounts for actually written bytes, and for buffered bytes.
varchar writers (and eg maps) are ready for next stripe, so they think there will be non-zero bytes to be written right after flushStripe completes